docs/adr/0009-non-WMDE-release-notes.md

```diff
@@ -0,0 +1,37 @@
+# 9) Release notes for software not maintained by WMDE {#adr_0009}
+
+Date: 2021-01-15
+
+## Status
+
+proposed
+
+## Context
+
+We intend to package and release software that is is not maintained by WMDE. For example, the Wikidata Query Service (WDQS)
+
+This software comes from a variety of sources including software that is used and maintained by the WMF in Wikimedia
+production Wikis but also some from complete 3rd parties.
+
+Some of this software including ElasticSearch and WikibaseLocalMedia already have curated release notes for versions.
+
+Other software such asd WDQS and Mediawiki extensions do not have release notes. They may have notable changes documented either
```

Comment:
typo "asd"

Reply:
done (also why does my local dictionary think asd is a real word?)